lib/phoenix_live_view/controller.ex

```diff
@@ -19,7 +19,7 @@ defmodule Phoenix.LiveView.Controller do
   ## Examples
 
       defmodule ThermostatController do
-        ...
+        use AppWeb, :controller
```

Comment:
I believe the convention is to call those things `MyApp`.

```suggestion
        use MyAppWeb, :controller
```